FAQ.md

```diff
@@ -0,0 +1,52 @@
+# FAQ - Frequently Asked Questions
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+
+**Table of Contents**
+
+- [FAQ - Frequently Asked Questions](#faq---frequently-asked-questions)
+  - [Installation](#installation)
+  - [Trading](#trading)
+  - [Wallet](#wallet)
+  - [Contact us](#contact-us)
+
+<!-- markdown-toc end -->
+
+## Installation
+
+Q: Do I need to use Umbrel to use Itchysats?
+A: Not at all! While [Umbrel](https://getumbrel.com) is a convenient way to use ItchySats (one-click install from the Umbrel App Store), it is possible to try out itchysats with one of the following methods:
+
+- download the itchysats taker binary straight from [github releases page](https://github.com/itchysats/itchysats/releases).
+- use the official Itchysats [Docker container](https://github.com/itchysats/itchysats/pkgs/container/itchysats%2Ftaker)
+
+Q: I see a lot of binaries, which one should I choose?
+A: As an end-user, you might want to use `taker` application, as this allows to take CFD. `maker` is targeted towards market-makers who want to offer CFDs. The flavour of the application depends on the operating systems you use - we currently provide binaries for linux (x64 and ARM) and Darwin/macOS (Intel-based).
+
+Q: Do you support Windows?
+A: Whilst we do not build Windows binaries directly, you should still be able to use the official itchysats Docker container.
+
+## Trading
+
+Q: How long does a CFD last?
+A: CFDs opened on ItchySats expire after 24h, however every hour the contract is automatically extended for another hour.
```

Comment:
```suggestion
A: CFDs opened on ItchySats expire after 24h, however every hour the contract is automatically extended for another hour. This means you have a perpetual CFD that stays open until you decide to close it.
```

Reply:
Mentioning the `perpetual` here will help with the next question - perpetual comes out of nowhere there :)